best-practices/require-probes/good-pods.yaml

```diff
@@ -0,0 +1,56 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: goodpod01
+  labels:
+    app: myapp
+spec:
+  containers:
+  - name: busybox
+    image: busybox:1.35
+    startupProbe:
+      grpc:
+        port: 8888
+    readinessProbe:
+      tcpSocket:
+        port: 8080
+      periodSeconds: 10
+    livenessProbe:
+      tcpSocket:
+        port: 7070
+      periodSeconds: 20
```

Comment:
I would also just choose one probe here and not all 3 as it's unrealistic, but nevertheless still works.